Makefile

```diff
@@ -245,11 +245,8 @@ subscription-manager:
 	git -C subscription-manager checkout --quiet FETCH_HEAD
 	@echo "checked out subscription-manager/ ref $$(git -C subscription-manager rev-parse HEAD)"
 
-$(NODE_MODULES_TEST): package.json
-	# if it exists already, npm install won't update it; force that so that we always get up-to-date packages
-	rm -f package-lock.json
-	# unset NODE_ENV, skips devDependencies otherwise
-	env -u NODE_ENV npm install --ignore-scripts
-	env -u NODE_ENV npm prune
+FORCE:
```

Comment:
I think `make rpm` does not work becuase there is something wrong in the makefile, I can't figure out what it is.
It doesnt like the `FORCE` target and during rpmbuild in the `%install` phase it tries to run the `tools/node-modules` script which won't work because there is no `.git` repo.

I also debugged cockpit-files during rpmbuild and it does not try to run the script during its `%install` phase even when the `FORCE` target is the same in c-files makefile

Reply:
Figured it out, it was a mix of wrong dependencies in Makefile and a `rm -rf dist` in rpm spec file.